packages/autocomplete/src/utils.ts

```diff
@@ -18,3 +18,32 @@ export function searchAttrKey(content: string, cursor: number) {
   if (text.match(/(<\w+\s*)[^>]*$/) !== null)
     return text.match(/\S+(?=\s*=\s*["']?[^"']*$)/)?.[0]
 }
+
+/**
+ * fuzzy search
+ * eg: tct -> [t]ext-[c]en[t]er
+ * @param list
+ * @param keyword
+ * @returns
+ */
+export function searchFuzzy(list: string[], keyword: string) {
+  const regex = new RegExp(keyword.split('').join('.*'))
```

Comment:
Creating a new regex for every change does not sound very efficient to me. Can we use `fzf` or `Fuse.js` to do it?

Reply:
nice! Fzf can dot it better.